engineapi/core.py

```diff
@@ -128,6 +130,38 @@ def facet_cut(
     return transaction
 
 
+def crafting_gogogo(
```

Comment:
This code is not exercised in the tests. Not sure it needs to be, but just making you aware of this.